glean-core/src/database/mod.rs

```diff
@@ -27,12 +27,18 @@ mod backend {
     pub fn rkv_new(path: &Path) -> Result<Rkv, rkv::StoreError> {
         Rkv::new::<rkv::backend::Lmdb>(path)
     }
+
+    /// No migration necessary when staying with LMDB.
+    pub fn migrate(_path: &Path, _dst_env: &Rkv) {
+        // intentionally left empty
```

Comment:
```suggestion
        // Intentionally left empty.
```

Super nit. Feel free to ignore, but this makes it consistent with the rest of the comments in the codebase.